go.mod

```diff
@@ -4,60 +4,59 @@ go 1.13
 
 require (
 	github.com/apparentlymart/go-cidr v1.0.1
-	github.com/coreos/etcd v3.3.18+incompatible
 	github.com/joho/godotenv v1.3.0
 	github.com/kelseyhightower/envconfig v0.0.0-20180517194557-dd1402a4d99d
 	github.com/onsi/ginkgo v1.10.1
 	github.com/onsi/gomega v1.7.0
 	github.com/patrickmn/go-cache v0.0.0-20180815053127-5633e0862627
-	github.com/projectcalico/felix v0.0.0-20200530082554-d7efcf1ff34b
-	github.com/projectcalico/libcalico-go v1.7.2-0.20200529191439-77f762f742df
-	github.com/prometheus/client_golang v0.9.4 // indirect
+	github.com/projectcalico/felix v0.0.0-20200610220805-221c23d0a7dd
+	github.com/projectcalico/libcalico-go v1.7.2-0.20200610201441-7f5b8c620b5c
 	github.com/satori/go.uuid v1.2.0
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/pflag v1.0.5
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200401174654-e694b7bb0875
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
 	golang.org/x/mod v0.3.0 // indirect
 	golang.org/x/tools v0.0.0-20200601175630-2caf76543d99 // indirect
 	gopkg.in/yaml.v2 v2.3.0 // indirect
 	honnef.co/go/tools v0.0.1-2020.1.4 // indirect
 
 	// k8s.io/api v1.16.3 is at 16d7abae0d2a
-	k8s.io/api v0.0.0
+	k8s.io/api v0.17.2
 
 	// k8s.io/apimachinery 1.16.3 is at 72ed19daf4bb
-	k8s.io/apimachinery v0.0.0
+	k8s.io/apimachinery v0.17.2
 
 	// k8s.io/apiserver 1.16.3 is at 9ca1dc586682
-	k8s.io/apiserver v0.0.0
+	k8s.io/apiserver v0.17.2
```

Comment:
the comments are stale, make them one block, pls :+1:

Reply:
Right will do.